backend/app/services/config_service.py

```
@@ -0,0 +1,168 @@
+import json
+import sqlite3
+from typing import Optional
+from typing import List, Any, Dict
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)